README.md

```diff
@@ -1,6 +1,6 @@
 
 
-<h1 align="center"><img src="docs/images/logo-woocommerce.svg" width="300"><br>for iOS</h1>
+<h1 align="center"><img src="https://github.com/user-attachments/assets/67305452-eff1-48c0-b3b0-9dc3069c7e39" width="300"><br>for iOS</h1>
```

Comment:
minor np and not a blocker, I would prefer not to depend on Github for retrieving the image in the README; IMO keeping it as part of the repo like before was a safer approach and doesn't make depending on Github servers.

But this stays a personal taste 🙂

Reply:
Makes sense! Done: 29bf846aded86a7ef75d8d4dbe28db3c83d5392a